TheFourFifths/consus-client
# [TFF-41] Faults story

### Summary

We can log faults and remove them.

### Checklist


### How to Review

1. Run with the sister PR.
2. Make sure it works.

### Links

- [TFF-41](https://msoese.atlassian.net/browse/TFF-41)
- TheFourFifths/consus/pull/97
